### ISSUE TITLE
Add multiple Dodge/Parry penalty

### ISSUE THREAD

**Issue by HawkofMay** [sun-dragon-cult/fvtt-system-rqg#613]
Why: According to the rules, during a fight your dodge skill drops by 20% with each use (always having at least a 5% chance) but when used, the skill value remains the same. 

What: Having the skill drop automatically would be helpful and maybe a reset button for when the fight is over.

**Comment by wake42** [comment]
I agree this would be a nice feature. It think it should be part of a bigger combat tracker improvement case. 

### The penalty should
- apply to both dodge and weapon parry
- be increased every time a dodge or weapon parry is performed
- be cleared when moving to the next combat tracker round

### Other considerations
There might need to be a way to change / clear the penalty in case it gets out of sync (for example after an unintended dodge roll)

**Comment by wake42** [comment]
Part of a big bang PR (but not fully automated)
